### DISCUSSION TITLE
Find two hidden words

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#593] [Galaxy Trivia] [answered]
Discover the missing letters in the shaded squares and fill them in to make complete words, read horizontally. When the puzzle is complete, the letters in the shaded columns will spell two hidden words, reading from top to bottom.

<img width="271" alt="image" src="https://user-images.githubusercontent.com/5120290/233213155-b7a1391b-c108-4d77-bb79-100336a7c174.png">

**Comment by Veercodeprog** [comment]
Legislation
Stockbroker
Exterminate
Pessimistic
Reading from the shaded columns from top to bottom, the hidden words are "listen" and "insect".
If we take the first shaded column, we can see that the letters are "L", "S", "E", and "P". Reading these letters from top to bottom, we get the word "LISP".

**Comment by ParanoidUser** [comment]
**Hint:** Both hidden words are in plural.

**Comment by peterbaccaro** [comment, marked as answer]
LEGISLATION
STOCKBROKER
EXTERMINATE
PESSIMISTIC

The hidden words are **ICES** and **TONS**.
